src/stylesheets/components/_rating.scss

```diff
@@ -0,0 +1,24 @@
+.rating {
+  margin-top: rem(45px);
+
+  text-align: center;
+
+  &__radio {
+    display: none;
+  }
+
+  &__emoji {
+    font-size: map-get($font-sizes, 'xxl');
+    line-height: rem(34px);
+    letter-spacing: rem(10px);
```

Comment:
@rafayet-monon How about change this to `16px`?
![Screen Shot 2563-12-03 at 11 00 15](https://user-images.githubusercontent.com/1772999/100962378-cac06c80-3556-11eb-804b-45a840f86fa6.png)